frame/contracts/src/lib.rs

```diff
@@ -123,6 +123,7 @@ use pallet_contracts_primitives::{
 	StorageDeposit,
 };
 use scale_info::TypeInfo;
+use smallvec::Array;
```

Comment:
Why do we need this import? It seems unnecessary.

Reply:
this trait is needed to be in scope in order to use `T::CallStack::size()`

Reply:
I see. Then you can also change `smallvec::Array` to `Array` in the `Config` trait now that we have it in scope.